gravitee-apim-console-webui/src/management/api/proxy/failover/api-proxy-failover.component.html

```diff
@@ -0,0 +1,65 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<h1>Failover</h1>
+
+<form *ngIf="failoverForm" [formGroup]="failoverForm" autocomplete="off" gioFormFocusInvalid>
+  <mat-card>
+    <mat-card-content>
+      <!-- Enable failover -->
+      <gio-form-slide-toggle appearance="fill" class="failover-card__enable-toggle">
+        <gio-form-label>Enable Failover</gio-form-label>
+        When gateway receives a connect timeout from backend servers, it redirects traffic to the next server until "max attempts" or
+        "timeout" have been reached.
+        <mat-slide-toggle
+          gioFormSlideToggle
+          formControlName="enabled"
+          aria-label="Failover enable toggle "
+          name="enableFailover"
+        ></mat-slide-toggle>
+      </gio-form-slide-toggle>
+
+      <mat-divider></mat-divider>
+
+      <!-- Max Attempts -->
+      <div class="failover-card__max-attempts" [class.disabled]="!enabled.value">
+        <div class="failover-card__max-attempts__label">Max attempts</div>
+        <p class="failover-card__max-attempts__hint">
+          Limits the number of possible tries before returning an error. Each try get an endpoint according to the load balancing algorithm.
+        </p>
+        <mat-form-field class="failover-card__max-attempts__form-field" appearance="fill">
+          <mat-label>Max Attempts</mat-label>
+          <input matInput formControlName="maxAttempts" type="number" min="1" />
```

Comment:
What do you think to add required to have the `*` on the label ? 
And add `mat-error` if the required validator throw 

same for all required field.

Reply:
I added mat-error for fields.

I disagree with the * since in angular 13 they added stars to all required fields. If we will migrate in the near future, maybe it doesn't make much sense